.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+# Patch release pipeline
+#
+# This release pipeline does a patch release with the contents of the
+# branch that you select when running the workflow. 
```

Comment:
Unless I am mistaken, there is nothing in the workflow that ties this to _patch_ releases; it just runs on whatever is in the given branch, whether patch, major, or minor. I'd skip mentioning what type of release this is for to avoid confusion.

```suggestion
# Release pipeline
#
# This release pipeline does a release with the contents of the
# branch that you select when running the workflow. 
```